tools/modules/software/module_samba.sh

```
@@ -0,0 +1,133 @@
+	["module_samba,example"]="help install remove start stop enable disable configure"
+	["module_samba,desc"]="Samba setup and service setting."
+	["module_samba,status"]="Active"
+	["module_samba,doc_link"]="https://www.samba.org/samba/docs/"
+	["module_samba,group"]="Networking"
+	["module_samba,port"]="445"
+	["module_samba,arch"]="x86-64 arm64 armhf"
+)
+
+function module_samba() {
+	local title="samba"
+	local condition=$(which smbd)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)